SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -78,14 +79,19 @@
         "nss_wrapper",
         "oath-toolkit",
         "open-vm-tools",
+        "opensc",
         "openssl",
         "pam_wrapper",
         "p7zip",
         "p11-kit",
+        "pcsc-lite",
+        "pcsc-lite-ccid",
         "perl-App-cpanminus",
         "perl-CPAN-DistnameInfo",
         "perl-CPAN-Meta-Check",
+        "perl-Fedora-VSP",
```

Comment:
Update: I've taken your `perl-generators` and `perl-Fedora-VSP` to unblock #1366. I've also applied the few clean-up comments in that PR, so once it's in, you can simply remove these two packages from here and use what's inside `dev`.